commit 1575b94a47ac6492e8f6aed8669377ce478cc3f3 (refs/changes/70/3770/3)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-04-03T20:04:38-07:00 (1 year, 6 months ago)
    
    TRITON-305 TRITON-262 should be more careful with async.waterfull arguments
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 1195d59..14baf07 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # sdc-config-agent changelog
 
+## 1.7.1
+
+- [TRITON-305] Fix breakage in 1.7.0 that breaks config-agents in zones.
+
 ## 1.7.0
 
 - [TRITON-262] Add support for `auto.INSTANCE_UUID`
diff --git a/agent.js b/agent.js
index 96609ff..0efe864 100644
--- a/agent.js
+++ b/agent.js
@@ -275,7 +275,9 @@ function setInZoneAutoMetadata(callback) {
 				});
 			}
 		]
-	}, callback);
+	}, function doneInZoneAutoMetadata(err) {
+		callback(err);
+	});
 }
 
 async.waterfall([
diff --git a/package.json b/package.json
index ecbc2eb..5a00fc0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "config-agent",
     "description": "SmartDataCenter Config Agent",
-    "version": "1.7.0",
+    "version": "1.7.1",
     "author": "Joyent (joyent.com)",
     "dependencies": {
         "assert-plus": "0.1.5",
